### PR TITLE
Parse operator

### PR DESCRIPTION
**Pull request by irv-fs**
This operator will allow us to deal with strings that need to be parsed by some delimiter.  Made sure it would work with adobe's standard encoding.

### CODE REVIEW

File: docs/operator_parse.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+
+## Usage
+
+## Data examples
+To simplify understanding, some simple parsing examples will be placed here to show what the output would be of the operator.  Many real world examples would only be interested in one property of the parsed values, and this section will give more details on what is flowing through the operator.  They will start with a `data` value as input to the operator, then show the output the operator would produce.
+> **Note:** These Data Examples are not full DLO rules.  They are meant to show the input and output of the parse operator under various circumstances. 
+
+### Simple Parsing
+Many uses of the `parse` operator will just want to parse a property based on one delimiter.  That is readily supported
```
Comment (djwhatley-fs) [inline docs/operator_parse.md:25]: Nit: missing punctuation at the end of this line

File: docs/operator_parse.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+  ],
+  "destination": "FS.event"
+}
+```
+
+### Input
+
+```javascript
+ s {
```
Comment (djwhatley-fs) [inline docs/operator_parse.md:212]: This and the example below have an extra space at the beginning that isn't present in the first one.

**Review by djwhatley-fs — APPROVED** [review]
LGTM - just a couple nits in the markdown

File: docs/operator_parse.md (hunk 2 above)
Comment (irv-fs) [reply docs/operator_parse.md]: Removed extra space

**Comment by djwhatley-fs** [comment]
@irv-fs Is this ready to merge?

**Comment by irv-fs** [comment]
> @irv-fs Is this ready to merge?

Yes, it is from my perspective.

**Pull request merged**